client/scss/utilities/_variables.scss

```diff
@@ -255,7 +262,7 @@ $breakpoints: (
   'small': 0,
   'medium': 600px,
   'large': 960px,
-  'xlarge': 1424px
+  'xlarge': $container-width-max
```

Comment:
Would it be worth thinking about naming these `tablet` etc - something more descriptive?